src/Core/HLSLCompiler/Compiler.cpp

```diff
@@ -100,9 +100,9 @@ std::vector<uint8_t> Compile(const ShaderDesc& shader, ShaderBlobType blob_type)
     if (blob_type == ShaderBlobType::kSPIRV)
     {
         arguments.emplace_back(L"-spirv");
-        arguments.emplace_back(L"-fspv-target-env=vulkan1.2");
+        arguments.emplace_back(L"-fspv-target-env=vulkan1.3");
         arguments.emplace_back(L"-fspv-extension=KHR");
-        arguments.emplace_back(L"-fspv-extension=SPV_NV_mesh_shader");
+        arguments.emplace_back(L"-fspv-extension=SPV_EXT_mesh_shader");
```

Comment:
Unfortunately, DirectXShaderCompiler do not support SPV_EXT_mesh_shader.

Reply:
This feature need the lastest dxc. The one bundled with vulkan sdk 1.3.231 is OK.

Reply:
You are right.
I think this pr should remain pending.